docs/source/conf.py

```diff
@@ -1,26 +1,258 @@
+#!/usr/bin/env python
+#
+# pyesmda documentation build configuration file, created by
+# sphinx-quickstart on Fri Jun  9 13:47:02 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another
+# directory, add these directories to sys.path here. If the directory is
+# relative to the documentation root, use os.path.abspath to make it
+# absolute, like shown here.
+
+import datetime
+
+# FIX: https://github.com/mgaitan/sphinxcontrib-mermaid/issues/72
+import errno
+import os
+import sys
 from subprocess import check_output
 
+import sphinx.util.osutil
+from sphinx.ext.napoleon.docstring import GoogleDocstring
```

Comment:
We have decided on NumPy docstrings, so I think all references to Google docstrings can go away.

Reply:
These stuff were simply to rename the "Variable" section to "Attributes" but since you think exposing attributes might not be a good idea, this is not required.